rcp/org.testeditor.web.testexecutionlogview/login_test/css/style.css

```diff
@@ -0,0 +1,144 @@
+@import url(http://fonts.googleapis.com/css?family=Vibur);
```

Comment:
That would be possible, but i changed some parts of the original one to serve our needs. So the result would not be the same.